examples/predicate-fields/README.md

```diff
@@ -74,114 +77,142 @@ $ docker run --detach --name vespa --hostname vespa-container \
   vespaengine/vespa
 </pre>
 
-Verify that configuration service (deploy api) is ready
+Verify that configuration service (deploy api) is ready:
 
 <pre data-test="exec">
 $ vespa status deploy --wait 300
 </pre>
 
-Download this sample application
+Download this sample application:
 
 <pre data-test="exec">
 $ vespa clone examples/predicate-fields my-app && cd my-app
 </pre>
 
-Build the sample app 
+Build the sample app:
 
 <pre data-test="exec">
 $ mvn package -U 
 </pre>
 
-Finally, we can deploy the app
+Finally, deploy the app:
 
 <pre data-test="exec">
 $ vespa deploy 
 </pre>
 
 ## Index users  
-This example use the [vespa-feed-client](https://docs.vespa.ai/en/vespa-feed-client.html) to feed
-documents. The users in the market place. 
+This example uses the [vespa-feed-client](https://docs.vespa.ai/en/vespa-feed-client.html) to feed documents.
+The users in the marketplace:
 ```json lines
 {"put": "id:s:user::alice", "fields": {"target": "gender in ['male'] and age in [30..40] and income in [200..50000]", "age": 23, "gender": ["female"]}}
```

Comment:
I suggest we use a more realistic range like [20000..50000]

Reply:
I was thinking it represented K dollars :)